src/main/kotlin/no/nav/syfo/varsel/arbeidstaker/ArbeidstakerVarselApi.kt

```diff
@@ -42,6 +44,59 @@ fun Route.registerArbeidstakerVarselApi(
             }
         }
 
+        get("/{$arbeidstakerVarselApiVarselParam}$arbeidstakerVarselApiPdfPath") {
+            val callId = getCallId()
+            try {
+                val requestPersonIdent = call.personIdent()
```

Comment:
Herfra og ned til hasAccessToVarsel-sjekken på linje 77 er ganske identisk med tjenesten under så du kan jo vurdere om noe av det kan refaktoreres ut til egne hjelpemetoder.

Reply:
Ja, jeg har sett litt på det. Her er det forbedringspotensiale, men det er ikke helt trivielt. Det ville antakelig hjelpe å innføre et interface/abstrakt superklasse for varslene (eller brev, eller hva de skal hete). Vil gjerne gjøre det i en egen PR.